app/src/main/java/io/intrepid/contest/models/InvitationResponse.java

```diff
@@ -0,0 +1,17 @@
+package io.intrepid.contest.models;
+
+import com.google.gson.annotations.Expose;
+
+import java.util.Date;
+import java.util.UUID;
+
+public class InvitationResponse {
+    @Expose
+    public UUID id;
+    @Expose
+    public UUID contestantId;
```

Comment:
These three arent used. Typically we would remove and add them back if needed but I'll let the two of you make the call on that.